src/stream/stream_backend.rs

```diff
@@ -191,6 +205,9 @@ fn create_udp_stream(
                 )));
             }
         },
+        VideoSourceType::Redirect(redirect_source) => match &redirect_source.source {
```

Comment:
why is this here ? create stream will not be called with redirect

Reply:
OOoh, you found it, I initially did the redirect inside this function and I had forgotten to remove it :facepalm: